sdk/python/feast/infra/registry/registry.py

```diff
@@ -338,10 +338,27 @@ def list_data_sources(
     def apply_data_source(
         self, data_source: DataSource, project: str, commit: bool = True
     ):
+        now = _utc_now()
+        if not data_source.created_timestamp:
+            data_source.created_timestamp = now
+        data_source.last_updated_timestamp = now
```

Comment:
Should the update time be updated just for registering the data source in registry. I dont see it is an actual update.

wdyt ?